src/vivarium_nih_us_cvd/components/effects.py

```diff
@@ -67,8 +81,26 @@ def _adjust_target(
                 primary_non_adherent,
                 choices=list(data_values.OUTREACH_EFFECTS[medication_type].keys()),
                 p=list(data_values.OUTREACH_EFFECTS[medication_type].values()),
-                additional_key=f"adjust_{medication_type}_medication_adherence_values",
+                additional_key=f"outreach_adjust_{medication_type}_medication_adherence_values",
             )
             target[primary_non_adherent] = adjusted
 
         return target
+
+    def _polypill_sbp_adherence_modifier(
```

Comment:
This layer of indirection is not needed since `_polypill_sbp_adherence_modifier` is identical to `_polypill_adjust_target`